account_consolidation/wizard/consolidation_consolidate.py

```diff
@@ -155,11 +155,11 @@ def reverse_moves(self, subsidiary):
             [('journal_id', '=', self.journal_id.id),
              ('to_be_reversed', '=', True),
              ('consol_company_id', '=', subsidiary.id)])
-        reversal_action = self.env['account.move.reverse'].with_context(
+        reversal_action = self.env['account.move.reversal'].with_context(
```

Comment:
There's an issue here with the moves to reverse (I'm not sure if it's only on v12.0 so maybe it's worth it to write a test and backport it to 11.0 to check if we have a bug)

Here's what I did:
1. Run a consolidation on August (ok)
2. Post the consolidation moves from August (ok)
3. Run a consolidation on October, August moves are reversed on the first day of October (ok)
4. Run a consolidation on December without posting the moves from October. (ko)
![image](https://user-images.githubusercontent.com/778828/58186566-724bb600-7cb5-11e9-94fb-3c8e8a3f87e0.png)

On point 4, all the moves from the previous consolidation are reversed, however:
- Moves from the August consolidation are reversed a second time (they should not as they were already reversed with the consolidation from October)
- Moves from the October consolidation are reversed and posted although they were not posted (it's correct to reverse but they should stay unposted)